core/src/main/java/org/apache/carbondata/core/locks/LocalFileLock.java

```diff
@@ -90,8 +90,9 @@ public LocalFileLock(String lockFileLocation, String lockFile) {
    * @param tableIdentifier
    * @param lockFile
    */
-  public LocalFileLock(CarbonTableIdentifier tableIdentifier, String lockFile) {
-    this(tableIdentifier.getDatabaseName() + CarbonCommonConstants.FILE_SEPARATOR + tableIdentifier
+  public LocalFileLock(AbsoluteTableIdentifier tableIdentifier, String lockFile) {
+    this(tableIdentifier.getCarbonTableIdentifier().getDatabaseName()
```

Comment:
Don't use the storelocation  to create path, use tablepath

Reply:
Fixed